mkosi/installer/__init__.py

```diff
@@ -47,6 +47,13 @@ def finalize_environment(cls, context: Context) -> dict[str, str]:
             context.config.bootable != ConfigFeature.disabled
         ):
             env["KERNEL_INSTALL_BYPASS"] = "1"
+        else:
+            env |= {
+                "SYSTEMD_XBOOTLDR_PATH": "/boot",
+                "SYSTEMD_ESP_PATH": "/efi",
+                # Required to make 90-loaderentry.install put the right paths into the bootloader entry.
+                "BOOT_MNT": "/boot",
```

Comment:
Instead of setting all these variables, shouldn't we just set `BOOT_ROOT`? That seems more in line what is actually advertised.

Reply:
We will still need to set `$BOOT_MNT` regardless unfortunately.